src/containers/BooksList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+const mapStateToProps = (state) => ({
+  state,
+});
+
+const mapDispatchToProps = () => ({});
+
+const renderList = ({
+  state,
+}) => (
+  <div className="table-wrapper">
+    <table className="fl-table">
+      <thead>
+        <th>BookId</th>
+        <th>Category</th>
+        <th>Title</th>
+      </thead>
+      <tbody>
+        {
+                    state.books.map((book) => (
+                      <Book
+                        key={book.id}
+                        // eslint-disable-next-line react/jsx-props-no-spreading
+                        {...book}
```

Comment:
- [ ] it passes a `book` object as a prop to each `Book` component (I agree that you are passing the book but it should be passed as a prop to the component, something like this: book={book})